frontend/src/metabase/query_builder/actions/core/core.js

```diff
@@ -255,6 +256,7 @@ export const apiUpdateQuestion = (question, { rerunQuery } = {}) => {
           question,
           originalQuestion,
         ),
+        excludeVisualisationSettings: isMetric,
```

Comment:
I just realized there is no way to change metric's viz.
Started a thread about it: https://metaboat.slack.com/archives/C051AG38B2S/p1728905762015299

This probably should not block this PR though.